docs/notebooks/icnn_inits.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>is this used? ctrl+F can't find it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Removed.</p>